extensions/wikia/AppPromoLanding/AppPromoLandingController.class.php

```diff
@@ -308,17 +308,18 @@ private function getIosUrl( $config ){
 	 * to be done by a specific Oasis module.  However, instead of being located inside of
 	 * a special page, we are doing this by stealing the "Community_App" article title.
 	 */
-	public static function onOutputPageBeforeHTML( OutputPage &$out, &$text ){
-		// Only steal this page if the wiki has an app configured.
-		$config = AppPromoLandingController::getConfigForWiki( F::app()->wg->CityId );
-		if($config !== null){
-			$title = $out->getTitle();
-			$origTitle = $title->getDBkey();
-			
-			if($origTitle == self::PROMO_PAGE_TITLE){
+	public static function onOutputPageBeforeHTML( OutputPage &$out, &$text ) {
+		$title = $out->getTitle();
+		$origTitle = $title->getDBkey();
+
+		if( $origTitle === self::PROMO_PAGE_TITLE ){
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)